components/CharacterCard.vue

```diff
@@ -0,0 +1,50 @@
+<script lang="ts" setup>
+
+defineProps({
```

Comment:
`defineProps<{ character: Character; }>()` this is how typed props should look in Vue 3. Type Object doesn't tell you information about fields character contains